### DISCUSSION TITLE
Support of arm-arch64 architecture

### DISCUSSION THREAD

**Discussion by drjh** [valentjn/vscode-ltex#890] [General]
Hi there,
as I understood this extension uses an encapsulated JVM. Is there a way to use a arm-arch64 based JVM, as the x64 JVM consumes a huge amount of power during emulation on a M1 processor.
Sometimes the extensions seems to hang and consumes almost all the CPU time, maybe most for emulation.

**Comment by spitzerd** [comment]
LTeX+ supports ARM
See https://github.com/ltex-plus/ltex-ls-plus/releases  and https://github.com/ltex-plus/vscode-ltex-plus
